### PR TITLE
[release-0.58] [e2e] Fix flaky tls test

### PR DESCRIPTION
**Pull request by fossedihelm**
**What this PR does / why we need it**:
This is a manual backport of https://github.com/kubevirt/kubevirt/pull/9195
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

/cc @enp0s3

### CODE REVIEW

**Comment by xpivarc** [comment]
/hold
Just to merge to main first

**Review by enp0s3 — APPROVED** [review]
/approve

File: tests/migration_test.go [hunk 1]
```diff
@@ -340,6 +340,13 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		return kvc.Spec.Configuration
 	}
 
+	getVmisNamespace := func(vmi *v1.VirtualMachineInstance) *k8sv1.Namespace {
+		namespace, err := virtClient.CoreV1().Namespaces().Get(context.Background(), vmi.Namespace, metav1.GetOptions{})
+		Expect(err).ShouldNot(HaveOccurred())
```
Comment (enp0s3) [inline tests/migration_test.go:345]: @fossedihelm I think its better to use `expectWithOffset` here

Comment (fossedihelm) [reply tests/migration_test.go]: Thanks, done!

**Comment by enp0s3** [comment]
/unhold
main pr merged

**Comment by enp0s3** [comment]
@fossedihelm Hi, can you PTAL at the ci failures? I think its related to the changes

**Pull request merged**